vnext/Microsoft.ReactNative/Fabric/platform/react/renderer/graphics/Color.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Color.h"
+#include <Utils/ValueUtils.h>
+
+namespace facebook {
+namespace react {
+
+xaml::Media::Brush SharedColor::AsWindowsBrush() const {
+  if (!m_color)
+    return nullptr;
+  if (!m_color->m_platformColor.empty()) {
+    return Microsoft::ReactNative::BrushFromColorObject(winrt::to_hstring(m_color->m_platformColor));
+  }
+  return xaml::Media::SolidColorBrush(m_color->m_color);
+}
+
+SharedColor colorFromComponents(ColorComponents components) {
```

Comment:
these can all be constexpr

Reply:
I'm not sure I follow.  `components` isn't a constant.

Reply:
the functions themselves I mean :) (assuming ColorComponents is a POD)

Reply:
ColorComponents is, but SharedColor is not.